{
  "comments": [
    {
      "key": {
        "uuid": "23b7830c_adaf987e",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 17,
      "author": {
        "id": 1079148
      },
      "writtenOn": "2018-11-28T16:55:24Z",
      "side": 1,
      "message": "Huh. That seems weird...\n\nFiled b/120135307 to look into this.",
      "range": {
        "startLine": 15,
        "startChar": 53,
        "endLine": 17,
        "endChar": 40
      },
      "revId": "8ac2f27cc2fa5fbfc1bbd1cede906d254a77f862",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "675f6d11_70632d27",
        "filename": "/COMMIT_MSG",
        "patchSetId": 2
      },
      "lineNbr": 17,
      "author": {
        "id": 1129218
      },
      "writtenOn": "2018-11-29T02:29:43Z",
      "side": 1,
      "message": "Thanks a lot for Josh, maybe my description is not very clear:)\nIt is meaning that the prerequisite for reproduce this problem, we need to make sure that zygote64 process has not been killed or restarted before.\n\nSorry, I could not access this issue 120135307.",
      "parentUuid": "23b7830c_adaf987e",
      "range": {
        "startLine": 15,
        "startChar": 53,
        "endLine": 17,
        "endChar": 40
      },
      "revId": "8ac2f27cc2fa5fbfc1bbd1cede906d254a77f862",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}